packages/postgres-database/src/simcore_postgres_database/constants.py

```diff
@@ -0,0 +1,6 @@
+from decimal import Decimal
+from typing import Final
+
+# NOTE: this is sync with DECIMAL_PLACES@ packages/models-library/src/models_library/basic_types.py using test_postgres_and_models_library_same_decimal_places_constant
+DECIMAL_PLACES: Final = 2
+QUANTIZE_EXP_ARG: Final = Decimal(f"1e-{DECIMAL_PLACES}")
```

Comment:
What does this mean? QUANTIZE_EXP_ARG not clear from the name to me?

Reply:
same for me. what does that mean?

Reply:
ok I checked here [https://docs.python.org/3/library/decimal.html](https://docs.python.org/3/library/decimal.html)

Reply:
documented